data-reporting-service/person-service/src/main/java/gov/cdc/etldatapipeline/person/model/dto/patient/PatientSp.java

```diff
@@ -7,19 +7,23 @@
 import jakarta.persistence.Column;
 import jakarta.persistence.Entity;
 import jakarta.persistence.Id;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
 import lombok.Data;
 import lombok.NoArgsConstructor;
 import lombok.extern.slf4j.Slf4j;
 
 @Slf4j
 @Data
+@Builder
 @Entity
 @NoArgsConstructor
+@AllArgsConstructor
 @JsonIgnoreProperties(ignoreUnknown = true)
-public class Patient {
+public class PatientSp {
```

Comment:
Could we add a comment in a later commit or so at the start of the file to add some context to what this dto is for?

Reply:
That being said, I like the separation of output objects

Reply:
Sure thing. Will do.